e2e/cypress/support/types.ts

```diff
@@ -24,17 +41,17 @@ export type RuleType = 'NODE' | 'EVENT';
 export type ConditionOperator = 'MEMBER_OF' | 'EQUALS';
 
 export interface Rule {
-  id?: string;
-  project_id?: string;
+  id: string;
```

Comment:
what was the `?` doin?

Reply:
it was making the fields optional and I couldn't tell why, especially since those are the required fields of a rule